Mathlib/RingTheory/Polynomial/Wronskian.lean

```diff
@@ -114,21 +114,12 @@ theorem natDegree_wronskian_lt_add {a b : R[X]} (hw : wronskian a b ≠ 0) :
   · exact Polynomial.degree_eq_natDegree ha
   · exact Polynomial.degree_eq_natDegree hb
 
-variable {k : Type*} [Field k]
-
-@[simp]
-theorem dvd_derivative_iff {a : k[X]} : a ∣ derivative a ↔ derivative a = 0 where
-  mp h := by
-    by_cases a_nz : a = 0
-    · simp only [a_nz, derivative_zero]
-    exact eq_zero_of_dvd_of_degree_lt h (degree_derivative_lt a_nz)
-  mpr h := by simp [h]
-
 /--
 For coprime polynomials `a` and `b`, their Wronskian is zero
 if and only if their derivatives are zeros.
 -/
-theorem _root_.IsCoprime.wronskian_eq_zero_iff {a b : k[X]} (hc : IsCoprime a b) :
+theorem _root_.IsCoprime.wronskian_eq_zero_iff
+    [NoZeroDivisors R] {a b : R[X]} (hc : IsCoprime a b) :
```

Comment:
```suggestion
    [NoZeroDivisors R] {P Q : R[X]} (hc : IsCoprime a b) :
```
Maybe I am too traditionalist.

Reply:
I reverted this one because this file consistently uses `a` and `b`

Reply:
OK, no problem.